src/wallet/rpcwallet.cpp

```diff
@@ -687,6 +687,8 @@ UniValue getbalance(const JSONRPCRequest& request)
     if (request.params.size() == 0)
         return  ValueFromAmount(pwalletMain->GetBalance());
 
+    const string* account = request.params[0].get_str() != "*" ? &request.params[0].get_str() : nullptr;
```

Comment:
I think this results in no longer throwing `RPC_WALLET_INVALID_ACCOUNT_NAME` when someone tries to get balance of `"*"`, instead, it returns back the total balance. But I think this is what we want....